### PR TITLE
Strict rules for cs fixer

### PR DESCRIPTION
**Pull request by chachanagov**

### CODE REVIEW

File: src/Client/TrapHandle/ContextProvider/Source.php [hunk 1]
```diff
@@ -17,8 +16,8 @@
  * @author Nicolas Grekas <p@tchwork.com>
  * @author Maxime Steinhausser <maxime.steinhausser@gmail.com>
  *
- * @link https://github.com/symfony/var-dumper/blob/7.0/Dumper/ContextProvider/SourceContextProvider.php
- * @link https://github.com/symfony/var-dumper/blob/6.3/Dumper/ContextProvider/SourceContextProvider.php
+ * @see https://github.com/symfony/var-dumper/blob/7.0/Dumper/ContextProvider/SourceContextProvider.php
+ * @see https://github.com/symfony/var-dumper/blob/6.3/Dumper/ContextProvider/SourceContextProvider.php
```
Comment (roxblnfk) [inline src/Client/TrapHandle/ContextProvider/Source.php:20]: yes, `@see` may contain a web page link but we use `@link` for that. It's semantically closer

File: src/Command/Test.php [hunk 2]
```diff
@@ -180,23 +185,22 @@ private function mail(OutputInterface $output, bool $multipart = false): void
...
-        Socket $socket,
+        \Socket $socket,
         string $content,
         string $expectedResponsePrefix,
     ): void {
         if ($content !== '') {
             \socket_write($socket, $content);
             // print green "hello" string in raw console markup
             $output->write(
-                '> ' . \str_replace(["\r", "\n"], ["\e[32m\\r\e[0m", "\e[32m\\n\e[0m"], $content),
```
Comment (roxblnfk) [inline src/Command/Test.php:199]: The previous linter added spaces here. Now this removes ones :D

File: src/Config/Server/Frontend.php [hunk 3]
```diff
@@ -4,16 +4,18 @@
...
 
 /**
  * @internal
  */
 final class Frontend
 {
-    /** @var int<1, 65535> */
+    /**
+     * @var int<1, 65535>
+     */
```
Comment (roxblnfk) [inline src/Config/Server/Frontend.php:18]: Is it possible to allow inline type hint for class properties?

File: src/Handler/Http/Middleware/SentryTrap/EnvelopeParser.php [hunk 4]
```diff
@@ -75,7 +74,8 @@ private static function parseItem(StreamInterface $stream): EnvelopeItem
 
     /**
      * @param positive-int $possibleBytes Maximum number of bytes to read. If the read fragment is longer than this
-     *        an exception will be thrown. Default is 10MB
+     *                                    an exception will be thrown. Default is 10MB
+     *
```
Comment (roxblnfk) [inline src/Handler/Http/Middleware/SentryTrap/EnvelopeParser.php:78]: We align comments with the word `@param`

Comment (chachanagov) [reply src/Handler/Http/Middleware/SentryTrap/EnvelopeParser.php:78]: This rule cannot be changed, only left or vertical

File: src/Client/TrapHandle/ContextProvider/Source.php (hunk 1 above)
Comment (chachanagov) [reply src/Client/TrapHandle/ContextProvider/Source.php]: Added a rule for @link

File: src/Command/Test.php (hunk 2 above)
Comment (chachanagov) [reply src/Command/Test.php:199]: Per 2 rule adds spaces, PhpCsFixer rule removes, which is better?

**Comment by roxblnfk** [comment]
Thank you. You helped compile the `ExtendedPERSet` rules list, which we can now use everywhere through the `wayofdev/cs-fixer-config` repository (#88)

**Pull request closed without merging**